packages/driver/src/multi-domain/events/logs.ts

```diff
@@ -0,0 +1,14 @@
+import { LogUtils } from '../../cypress/log'
+
```

Comment:
I would expect the primary LogManager to listen for when origin command log events to trigger updates to the runner's event-manager listeners. This would keep the logs generated in the cy.origin callback will be associated with the test, will keep the event flow the same to the runner's event-manager and will ensure the logGroups stay in sync from the primary manager.

Reply:
What's the primary LogManager?